content/200-concepts/100-components/02-prisma-client/030-crud.mdx

```diff
@@ -777,6 +777,70 @@ To resolve this error, you can:
   const transaction = await prisma.$transaction([deletePosts, deleteUser])
   ```
 
+### Delete all records from all tables
+
+Sometimes you want to remove all data from all tables but keep the actual tables. This can be particularly useful in a development environment.
+
+The following examples show how to remove all data (including relations) from all tables in your database, with `PostgreSQL`, `SQLite` and `MySQL`. The `SQLite` and `MySQL` examples use a `transaction` to batch the queries together so that they are executed on the same connection.
+
+<TabbedContent tabs={[<FileWithIcon text="PostgreSQL" icon="database"/>, <FileWithIcon text="SQLite" icon="database"/>, <FileWithIcon text="MySQL" icon="database"/>]}>
+
+<tab>
+
+```ts
+// Special fast path to drop data from a postgres database.
+// This is an optimization which is particularly crucial in a unit testing context.
+// This code path takes milliseconds, vs ~7 seconds for a migrate reset + db push
+for (const { tablename } of await this.prisma
+  .$queryRaw`SELECT tablename FROM pg_tables WHERE schemaname='${this.dbSchemaName}'`) {
+  await this.prisma.$queryRaw`TRUNCATE TABLE \"${this.dbSchemaName}\".\"${tablename}\" CASCADE;`
+}
+for (const { relname } of await this.prisma
+  .$queryRaw`SELECT c.relname FROM pg_class AS c JOIN pg_namespace AS n ON c.relnamespace = n.oid WHERE c.relkind='S' AND n.nspname='${this.dbSchemaName}';`) {
+  await this.prisma
+    .$queryRaw`ALTER SEQUENCE \"${this.dbSchemaName}\".\"${relname}\" RESTART WITH 1;`
+}
+```
+
+</tab>
+
+<tab>
+
+```ts
+const [numberOfRowsDeleted] = await prisma.$transaction([
+  prisma.$executeRaw`SET FOREIGN_KEY_CHECKS = 0;`,
+  prisma.$executeRaw`TRUNCATE TABLE teams;`,
+  prisma.$executeRaw`TRUNCATE TABLE players;`,
+  prisma.$executeRaw`SET FOREIGN_KEY_CHECKS = 1;`,
+])
+```
+
+</tab>
+
+<tab>
+
+```ts
+const [numberOfRowsDeleted] = await prisma.$transaction([
+  prisma.$executeRaw`SET FOREIGN_KEY_CHECKS = 0;`,
+  prisma.$executeRaw`TRUNCATE TABLE teams;`,
+  prisma.$executeRaw`TRUNCATE TABLE players;`,
+  prisma.$executeRaw`SET FOREIGN_KEY_CHECKS = 1;`,
+])
+```
+
+</tab>
+ 
+</TabbedContent>
+
+#### <inlinecode>Prisma Migrate</inlinecode>
+
+If you use Prisma Migrate, you can use `migrate reset`, this will:
+
+- Drop the database
+- Create a new database
+- Apply migrations
+- Seed the database with data
```

Comment:
iiuc, this is a sequence of steps, so I think it's a good opportunity to use a numbered instead of a bullet list:
```suggestion
1. Drop the database
1. Create a new database
1. Apply migrations
1. Seed the database with data
```

Reply:
Here's a quick overview when to use numbered and when to use unordered bullet lists:https://www.prisma.io/docs/about/style-guide#use-tables-bullet-lists-and-numbered-lists

![image](https://user-images.githubusercontent.com/4058327/113104065-6067c200-9200-11eb-8a25-56eabdc4f10d.png)